telemetryui/migrations/versions/466cf2f35d67_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: 466cf2f35d67
+Revises: 3230c615d6e0
+Create Date: 2017-09-12 21:33:00.335130
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '466cf2f35d67'
+down_revision = '3230c615d6e0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('classification_text_ind', table_name='classification')
+    op.add_column('records', sa.Column('bios_version', sa.String(), nullable=True))
+    op.add_column('records', sa.Column('board_name', sa.String(), nullable=True))
+    op.add_column('records', sa.Column('cpu_model', sa.String(), nullable=True))
+    op.alter_column('records', 'external',
```

Comment:
Strange, why is a migration needed for the 'external' field? Or, why did alembic add this call? (similar comment for the `downgrade` function below.

Reply:
Alembic must have added it bases on changes/queries.

Reply:
Hmm, do we remove it manually then? In the past, I have always modified alembic autogenerated code manually, because it sometimes makes too many changes.